2-mandatory/2-financial-times.js

```diff
@@ -5,7 +5,13 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
```

Comment:
Nice one 😄 
As an extra exercise, could you try re-writing this with the `filter` array method?